src/cunumeric/fft/fft.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2021 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "cunumeric/cunumeric.h"
+#include "cunumeric/fft/fft_util.h"
+
+namespace cunumeric {
+
+struct FFTArgs {
+  Array output;
+  Array input;
+  fftType type;
+  fftDirection direction;
+  bool operate_over_axes;
+  std::vector<int64_t> axes;
+};
+
+class FFTTask : public CuNumericTask<FFTTask> {
+ public:
+  static const int TASK_ID = CUNUMERIC_FFT;
+
+ public:
+  static void cpu_variant(legate::TaskContext& context);
```

Comment:
Please remove any variant that you didn't implement. Having an empty body would only make the mapper confused. The contract is that all variants must do the same computation.